src/redux/appSlice.js

```diff
@@ -0,0 +1,55 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import { httpService } from '../services/http-service';
+
+const _apiBase = 'https://api.themoviedb.org/3/';
+const _apiKey = 'api_key=a60262500ac52b0699a0d49e7f802ffa';
```

Comment:
as I see you are using these constants many times in app, lets move it to a separate file once and than just import them anywhere you need

Reply:
Fixed!